### ISSUE TITLE
Docker 2.1.6.0 update did not check OS Version

### ISSUE THREAD

**Issue by jboelter** [docker/for-win#5168]
### Expected behavior

The installer should check the OS version for compatibility before updating when in WSL2 mode.

### Actual behavior

Docker Desktop self-updated and did not check the underlying OS version for compatibility.  Have to revert back to 2.1.5.0.

![image](https://user-images.githubusercontent.com/3619564/69096017-680e3880-0a08-11ea-90d4-4d44b6df12e6.png)

### Information
  - Windows Version: Microsoft Windows [Version 10.0.19013.1122]
  - Docker Desktop Version: 2.1.6.0

Version that self-updated was Docker for Windows Edge 2.1.5.0
https://docs.docker.com/docker-for-windows/edge-release-notes/

edit: typo

**Comment by stoutlabs** [comment]
Same issue here. I'm on the slow ring, 19013...  and WSL2 + Docker is now broken.

No worries, I can just revert to the previous version – I just wanted to confirm the issue. 👍

**Comment by StefanScherer** [comment]
@jboelter @stoutlabs  Thanks for the feedback. The week before the Slow Ring release we got the 19018 release which unblocked a lot of improvements we had on hold for Docker Desktop. As we saw that Slow ring is got a release 19013 we decided against supporting both the old and new mode in one Docker Desktop release, as it's only a temporary problem. We apologize for the inconvenience. ;-)
Current workaround is to install 2.1.5.0 which you can download from https://docs.docker.com/docker-for-windows/edge-release-notes/ page.
We hope that Slow Ring will get an update to ≥ 19018 soon.

**Comment by StefanScherer** [comment]
We‘ve added a warning into the Known Issues section for Docker Desktop 2.1.6.1 for Slow Ring users. I‘m glad that Slow Ring now is on 19025, so everything is fine now.
